tests/tests/test_deployment_aborting.py

```diff
@@ -31,6 +31,7 @@ def abort_deployment(
         install_image,
         abort_step=None,
         mender_performs_reboot=False,
+        number_of_reboots=2,
```

Comment:
As discussed face to face, we can unify these two parameters and just interpret `number_of_reboots == 0` as `mender_performs_reboot == False`